sentences/nb/light_HassTurnOff.yaml

```
@@ -9,3 +9,8 @@ intents:
         slots:
           domain: light
           name: all
+        response: lights_area
+      - sentences:
+          - "<skru_på> <lys> <navn>"
```

I assume you meant skru_av: 
```suggestion
          - "<skru_av> <lys> <navn>"
```